config/system-test-driver.config.py

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env python
```

Comment:
This is an executable script. It feels quite odd being in a folder called config. Additionally, this folder has a stated purpose in the top-level directory to which this does not directly relate:
https://github.com/dcos/dcos/blob/master/README.md#repo-structure